app/src/androidTest/java/ch/epfl/sdp/blindly/profile_setup/TestProfileGender.kt

```diff
@@ -6,22 +6,27 @@ import androidx.test.espresso.assertion.ViewAssertions
 import androidx.test.espresso.intent.Intents
 import androidx.test.espresso.intent.Intents.intended
 import androidx.test.espresso.intent.Intents.times
+import androidx.test.espresso.intent.matcher.BundleMatchers
+import androidx.test.espresso.intent.matcher.IntentMatchers
 import androidx.test.espresso.intent.matcher.IntentMatchers.hasComponent
 import androidx.test.espresso.matcher.ViewMatchers
 import androidx.test.espresso.matcher.ViewMatchers.withId
 import androidx.test.ext.junit.rules.ActivityScenarioRule
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import ch.epfl.sdp.blindly.R
+
 import org.hamcrest.Matchers
 import org.junit.Rule
 import org.junit.Test
 import org.junit.runner.RunWith
 
+const val TEST_GENRE_MORE = "More"
+const val TEST_GENRE_MEN = "Man"
+const val TEST_GENRE_WOMEN = "Woman"
```

Comment:
I think these ones should be private too.

Reply:
Should they? They're actually shared in multiple tests classes (SettingsShowMeTest and SettingsTest for example). I'm not sure if it's better to copy them all over our tests or simply use the same evreywhere.

Reply:
My bad, I didn't see that they were used somewhere else. Better let them public !